CompilerSource/compiler/components/write_shader_data.cpp

```diff
@@ -16,26 +16,24 @@
 **/
 
 #include "makedir.h"
-#include <stdio.h>
-#include <iostream>
-#include <fstream>
-
-using namespace std;
-
 #include "syntax/syncheck.h"
 #include "parser/parser.h"
-
 #include "backend/GameData.h"
 #include "parser/object_storage.h"
 #include "compiler/compile_common.h"
+#include "languages/lang_CPP.h"
 
+#include <stdio.h>
+#include <iostream>
+#include <fstream>
+#include <vector>
```

Comment:
You don't need to include vector here, this source is not using vector. Remove all unused includes. You would want it to include vector in the generated header.

Reply:
I just checked this one because of #1607, and Josh/myself was not using vector in this source previously and it isn't now, so this remains unresolved. The vector include should be removed here.